src/stretch_ros2_bridge/stretch_ros2_bridge/remote/modules/mapping.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Hello Robot, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the LICENSE file in the root directory
+# of this source tree.
+#
+# Some code may be adapted from other open-source works with their respective licenses. Original
+# license information maybe found below, if so.
+
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from nav2_msgs.srv import LoadMap, SaveMap
+
+from .abstract import AbstractControlModule
+
+
+class StretchMappingClient(AbstractControlModule):
```

Comment:
how does this relate to the map saver https://github.com/hello-robot/stretch_ai/blob/main/src/stretch_ros2_bridge/stretch_ros2_bridge/ros/map_saver.py